src/app/public/modules/datepicker/datepicker.component.scss

```diff
@@ -2,10 +2,6 @@
 
 .sky-input-group-datepicker-btn {
```

Comment:
Does this class even need CSS rules now? Or does the button still have a border radius if not specified? If it does, that should probably be fixed in theme.

Reply:
Upgraded theme library. This line has now been removed.